digits/model/images/classification/views.py

```diff
@@ -265,9 +265,6 @@ def image_classification_model_classify_one():
     db_task = job.train_task().dataset.train_db_task()
     height = db_task.image_dims[0]
     width = db_task.image_dims[1]
-    if job.train_task().crop_size:
```

Comment:
I had to remove those lines to ensure consistency between methods: this seems to be the only place where images are being cropped before being fed to the DL framework (mean image, images in database and lists of images in classify_many are not cropped).